### PR TITLE
`selectExpr`

### PR DESCRIPTION
**Pull request by cafreeman**

### CODE REVIEW

File: pkg/R/DataFrame.R [hunk 1]
```diff
@@ -607,6 +607,37 @@ setMethod("select", signature(x = "DataFrame", col = "Column"),
             dataFrame(sdf)
           })
 
+
```
Comment (cafreeman) [inline pkg/R/DataFrame.R]: Just saw this extra space.

File: pkg/R/DataFrame.R [hunk 2]
```diff
@@ -607,6 +607,36 @@ setMethod("select", signature(x = "DataFrame", col = "Column"),
             dataFrame(sdf)
           })
 
+#' SelectExpr
+#'
+#' Select from a DataFrame using a set of SQL expressions.
+#'
+#' @param x A DataFrame to be sorted.
```
Comment (shivaram) [inline pkg/R/DataFrame.R]: `to be sorted` -> `to be selected from`

**Comment by shivaram** [comment]
LGTM - Had a minor doc comment

**Comment by cafreeman** [comment]
Fixed.

**Comment by shivaram** [comment]
Thanks. Merging this

**Pull request merged**